bench-vortex/src/datasets/public_bi_data.rs

```diff
@@ -521,12 +522,16 @@ impl PBIDataset {
             let compressed = idempotent_async(
                 &self.path_for_file_type(output_fname, FileType::Vortex),
                 |output_path| async {
-                    VortexWriteOptions::default()
+                    let mut f = VortexWriteOptions::default()
                         .write(
                             File::create(output_path).await.unwrap(),
                             parquet_to_vortex(f).await.unwrap(),
                         )
-                        .await
+                        .await?;
+
+                    f.flush().await?;
```

Comment:
Revert all the flushing plz

Reply:
https://github.com/spiraldb/vortex/pull/2798